funnel/templates/project.html.jinja2

```diff
@@ -69,13 +69,45 @@
     </div>
   </div>
 
-  <div class="grid about-ticket-wrapper">
-    <div class="{% if project.boxoffice_data and project.boxoffice_data.item_collection_id -%}grid__col-sm-6 grid__col-md-7 {% elif project.allow_rsvp -%}grid__col-sm-6{% else %}grid__col-sm-12{% endif %}" id="about">
-      <h2 class="mui--text-display1 mui--text-left project-section__headline">About</h2>
-      <div class="project-section__description">{{ project.description|safe }}</div>
+  <div class="about">
+    {% if project.boxoffice_data and project.boxoffice_data.item_collection_id or project.allow_rsvp -%}
+      <div class="about__participate">
+        <div class="mui--hidden-md mui--hidden-lg mui--hidden-xl about__participate__back-nav">
+          <div class="mobile-nav mui--z1">
+            <a href="javascript(0)" aria-label="Back to {{project.title}}" class="mui--text-dark mui--text-headline" id="close-ticket-widget"><i class="material-icons mui--align-middle mobile-nav__prev" aria-hidden="true">arrow_back</i></a> <span class="mui--text-dark mui--text-headline">Tickets</span>
```

Comment:
This `href` is wrong. Did you mean `javascript:0`? Ideally it should contain the actual URL.

Reply:
Use `aria-label="{% trans %}Back to project{% endtrans %}"`. I think there's another aria annotation to indicate that it's a back/close button.